frontend/src/modules/EditZing/Components/GroupCard.tsx

```diff
@@ -65,6 +68,14 @@ const GroupCard = ({
     setIsHovering(false)
   }
 
+  const removeGroup = (groupId: string, groupNumber: number) => {
+    axios.post(`${API_ROOT}${MATCHING_API}/hide-group`, {
+      courseId: courseId,
+      groupNumber: groupNumber,
+    })
+    window.location.reload() // this can be improved so page doesn't reloadA
+  }
+
```

Comment:
Yeah, please change this part. Every other operation that `POST`s to the Firestore database doesn't require a reload. Take a look at examples App.tsx for functions that show how to do this, as well as some other components that may update the database. You will probably have to work with React Context on this one.